rfcs/20220218-introduce_ui_state_cookie_to_frontend.md

```diff
@@ -0,0 +1,63 @@
+# RFC: Introduce UI state cookie to frontend
+
+- [x] @obulat 
+- [ ] Approver (TBD)
```

Comment:
@zackkrida based on https://github.com/WordPress/openverse-frontend/issues/909#issuecomment-1044871690 can I mark you as an approver?

```suggestion
- [x] @zackkrida 
```